lua/nvim-treesitter/install.lua

```diff
@@ -14,6 +14,7 @@ local lockfile = {}
 M.compilers = { vim.fn.getenv "CC", "cc", "gcc", "clang", "cl", "zig" }
 M.prefer_git = fn.has "win32" == 1
 M.command_extra_args = {}
+M.ts_generate_args = nil
```

Comment:
Do we need to make those public?

Reply:
Good question. Does it hurt? I may come in useful (this way you can override it easily, because when it's non-nil, it won't get touched further down).

Reply:
Huh indeed.